### ISSUE TITLE
Elements must have sufficient color contrast (.k-pager-info)

### ISSUE THREAD

**Issue by sryeluri** [telerik/kendo-ui-core#4852]
Below accessibility issue that needs investigation.

Issue Details
Elements must have sufficient color contrast
color-contrast

Element Path
.k-pager-info

Snippet
<span class="k-pager-info k-label">1 - 10 of 14 items</span>

Expected behaviour:
- Element has insufficient color contrast of 2.32 (foreground color: #aaaaaa, background color: #ffffff, font size: 11.3pt, font weight: normal). Expected contrast ratio of 4.5:1

To reproduce
Use K4W to investigate the issue details

Environment
Chrome version 70.0.3538.77
